scripts/mutant-timing.r

```diff
@@ -21,44 +26,62 @@ create_resident <- function(cache = F) {
 }
 
 # timing for resident only without caching
-system.time(create_resident())
+system.time(create_resident(p1))
 
 # timing for resident only with caching
-system.time(scm <- create_resident(cache = T))
+system.time(scm <- create_resident(p1, cache = T))
 
 # timing for mutant re-using node schedule
-s = FF16_Strategy()
-system.time(scm$run_mutant(list(s), append = F, update_schedule = F))
+system.time(scm$run_mutant(p1$strategies, append = F, update_schedule = F))
```

Comment:
for mutants it's better to add a prepared strategy than just modify the
default, as there's other parameters differing than just lma (generated via the hyperpaameter function, which runs when you call expand_parameters.